docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -119,26 +119,186 @@ that catalog name instead of ``sqlserver`` in the above examples.
 Type mapping
 ------------
 
-Trino supports the following SQL Server data types:
-
-==================================  ===============================
-SQL Server Type                     Trino Type
-==================================  ===============================
-``bigint``                          ``bigint``
-``tinyint``                         ``smallint``
-``smallint``                        ``smallint``
-``int``                             ``integer``
-``float``                           ``double``
-``char(n)``                         ``char(n)``
-``varchar(n)``                      ``varchar(n)``
-``date``                            ``date``
-``datetime2(n)``                    ``timestamp(n)``
-``datetimeoffset(n)``               ``timestamp(n) with time zone``
-==================================  ===============================
+Because Trino and SQL Server each support types that the other does not, this
+connector modifies some types when reading or writing data.
+
+SQL Server to Trino read type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This connector supports reading the following SQL Server types and performs
+conversion to Trino types with the detailed mappings as shown in the following
+table.
```

Comment:
See comment above. Would suggest "The connector maps SQL server types to the corresponding Trino types following this table:"